### PR TITLE
[FIX] analytic, account: use column_invisible instead of invisible

### PR DESCRIPTION
**Pull request by daiduongnguyen-odoo**
* When fwd https://github.com/odoo/odoo/pull/156349 , we forgot to adapt with change in >=17.0

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by duong77476-viindoo** [comment]
Hello @gawa-odoo can you take a look please ? Thank you

**Comment by duong77476-viindoo** [comment]
Hello @svfu-odoo Can you take a look please ?

File: addons/analytic/views/analytic_account_views.xml [hunk 1]
```diff
@@ -60,8 +60,8 @@
                     <field name="plan_id"/>
                     <field name="active" column_invisible="True"/>
                     <field name="company_id" groups="base.group_multi_company"/>
-                    <field name="debit" sum="Debit" invisible="1"/>
-                    <field name="credit" sum="Credit" invisible="1"/>
+                    <field name="debit" sum="Debit" column_invisible="1"/>
+                    <field name="credit" sum="Credit" column_invisible="1"/>
```
Comment (svfu-odoo) [inline addons/analytic/views/analytic_account_views.xml:64]: nitpick for consistency:
```suggestion
                    <field name="debit" sum="Debit" column_invisible="True"/>
                    <field name="credit" sum="Credit" column_invisible="True"/>
```

**Review by svfu-odoo — APPROVED** [review]
Thanks :slightly_smiling_face:
I left a small (optional to me) nitpick.
@JulienVR for final review

**Comment by gawa-odoo** [comment]
@robodoo r+

**Comment by duong77476-viindoo** [comment]
Runbot need sleeping i guess

**Comment by svfu-odoo** [comment]
@robodoo retry

**Pull request closed without merging**